packages/controllers/src/snaps/Timer.ts

```diff
@@ -1,57 +1,128 @@
-export class Timer {
-  private _timeout?: NodeJS.Timeout | number;
-
-  private _start?: number;
+import { assert } from '../utils';
 
-  private _remaining: number;
+export type Status = 'stopped' | 'paused' | 'running' | 'finished';
```

Comment:
We have a few things named `Status` now. We should probably rename this to `TimerStatus`.